### PR TITLE
Custom page settings support

### PR DESCRIPTION
**Pull request by ZaidRehman**

### CODE REVIEW

File: src/helper/theme.vue.config.ts [hunk 1]
```diff
@@ -86,9 +90,25 @@ const chainWebpack = (config) => {
     .optimization.splitChunks({
       automaticNameDelimiter: "_"
     })
+    config.module
```
Comment (ZaidRehman) [inline src/helper/theme.vue.config.ts]: Do we need this now? If not then remove it.

File: src/lib/Theme.ts [hunk 2]
```diff
@@ -34,7 +35,7 @@ import { downloadFile } from '../helper/download';
 import Env from './Env';
 import Debug from './Debug';
 import ora from 'ora';
-import { themeVueConfigTemplate } from '../helper/theme.vue.config';
+import { themeVueConfigTemplate,settingLoader } from '../helper/theme.vue.config';
```
Comment (ZaidRehman) [inline src/lib/Theme.ts]: No space after comma

File: src/lib/Theme.ts [hunk 3]
```diff
@@ -908,9 +903,13 @@ export default class Theme {
             theme.assets.umdJs = theme.assets.umdJs || {};
             theme.assets.umdJs.links = umdJsUrls;
             theme.assets.umdJs.link = "";
+            theme.assets.umd_js = theme.assets.umdJs || {};
```
Comment (ZaidRehman) [inline src/lib/Theme.ts:917]: Once confirm with @Shivraj0 about underscore issue

**Review by Shivraj0 — APPROVED** [review]
Approved

**Pull request merged**